mmvb_backend/case_synthesizer/api/urls.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.routers import DefaultRouter
+
+from case_synthesizer.api.views import CaseSynthesizerViewSet
+from case_synthesizer.apps import CaseSynthesizerConfig
+
+app_name = CaseSynthesizerConfig.name
+
+router = DefaultRouter(trailing_slash=False)
+router.register(
+    "case-synthesizer", CaseSynthesizerViewSet, basename="case-synthesizer",
```

Comment:
wild idea: use `/case-sets/synthesize` as the endpoint? 🤔

Reply:
I have mixed feelings about this. It would be good for having a more consistent api, but it would be bad for establishing a clear separation of contexts, since the case synthesizer is a separate application that won't probably even be active in a production environment. So the decision depends on what is more important among those options.

Reply:
i'm not sure whether your assumption about the production environment really holds – in the mid-term there will be a significant lack of test-data, so i think synthesising cases will remain important for a while. (for the scenario of private benchmarks, e.g. running a nightly test against your current development AI it might even be relevant in the long-term)
that, in combination with the idea to return a case-set URL rather than the actual data, would make me opt for the first path – and think about (re)moving it when the time comes

Reply:
The endpoint is actually `/api/v1/cases/synthesize` and it returns a list with the urls for the cases in the form `/api/v1/cases/<case_id>`.

Reply:
This is also a tricky bit, if we try to use `cases` or `case-sets` namespace in the path used for the case synthesizer it breaks with `POST method not allowed`. As an alternative we could use `/case/synthesize` if that is acceptable.